dash_table_base/__init__.py

```diff
@@ -76,7 +76,24 @@
         ).format(__version__),
         'namespace': package_name,
         'dynamic': True
+    },
+    {
+        'relative_package_path': 'async~highlight.js',
+        'external_url': (
+            'https://unpkg.com/dash-table@{}/dash_table/async~highlight.js'
+        ).format(__version__),
+        'namespace': package_name,
+        'async': True
+    },
+    {
+        'relative_package_path': 'async~highlight.js.map',
+        'external_url': (
+            'https://unpkg.com/dash-table@{}/dash_table/async~highlight.js.map'
+        ).format(__version__),
+        'namespace': package_name,
+        'async': True
```

Comment:
Could you update the table's init.py file so it uses the same pattern as other repos with multiple async entries? e.g. https://github.com/plotly/dash-core-components/blob/dev/dash_core_components_base/__init__.py#L42